sdk/storagecache/Azure.ResourceManager.StorageCache/src/Generated/AmlFilesystemCollection.cs

```diff
@@ -0,0 +1,341 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+// <auto-generated/>
+
+#nullable disable
+
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.Globalization;
+using System.Threading;
+using System.Threading.Tasks;
+using Azure;
+using Azure.Core;
+using Azure.Core.Pipeline;
+using Azure.ResourceManager;
+using Azure.ResourceManager.Resources;
+
+namespace Azure.ResourceManager.StorageCache
+{
+    /// <summary>
+    /// A class representing a collection of <see cref="AmlFilesystemResource" /> and their operations.
+    /// Each <see cref="AmlFilesystemResource" /> in the collection will belong to the same instance of <see cref="ResourceGroupResource" />.
+    /// To get an <see cref="AmlFilesystemCollection" /> instance call the GetAmlFilesystems method from an instance of <see cref="ResourceGroupResource" />.
+    /// </summary>
+    public partial class AmlFilesystemCollection : ArmCollection, IEnumerable<AmlFilesystemResource>, IAsyncEnumerable<AmlFilesystemResource>
```

Comment:
I think `Filesystem` should be `FileSystem`.
```suggestion
    public partial class AmlFileSystemCollection : ArmCollection, IEnumerable<AmlFileSystemResource>, IAsyncEnumerable<AmlFileSystemResource>
```

There are many places where `Filesystem` is used, which should be fixed. AFAICS, there are two options:
- fix the original swagger definitions
- or use batch renaming directive provided by us, see [here](https://github.com/Azure/azure-sdk-for-net/blob/c99853c16d6e167558e10dd56e2b98543c01617f/sdk/compute/Azure.ResourceManager.Compute/src/autorest.md?plain=1#L35-L49) for examples.

Reply:
@archerzz our swagger has already been merged with this naming convention: https://github.com/Azure/azure-rest-api-specs/pull/23563 . We had this reviewed and approved by ARM reviewers and our other generated SDKs follow this convention. I think "Filesystem" would be the better option?

Reply:
@rebecca337 I think it's `File System`, so in [.Net SDK guidelines](https://learn.microsoft.com/en-us/dotnet/standard/design-guidelines/names-of-type-members#names-of-properties), applying Pascal case will result in `FileSystem`.

Reply:
@archerzz I just fixed the naming for all AmlFilesystem references to AmlFileSystem